react-common/components/share/ShareInfo.tsx

```diff
@@ -146,6 +146,13 @@ export const ShareInfo = (props: ShareInfoProps) => {
         }
     }
 
+    const handleKioskHelpClick = () => {
+        pxt.log("this will lead the user to documentation");
+        // TODO: have this actually link to the documentation when it's done
+        //const kioskDocumentationUrl = ....
```

Comment:
It's usually easier to just point at a page here and fill that in later, since md is live / can be updated whenever and worst case is seeing a 404 -- occasionally you'll run into issues like cors or browsers not allowing pop ups for one reason or another so at least seeing it's going to a reasonable page is typically good :) (that said it looks like https://github.com/microsoft/pxt-arcade/pull/5514 is almost ready so easiest to just plug that in)